src/constants.ts

```diff
@@ -33,6 +33,9 @@ const BTC_ADDRESS_REGEX =
       BTC_REGTEST_REGEX;
 
 const PARACHAIN_URL = process.env.REACT_APP_PARACHAIN_URL || 'ws://127.0.0.1:9944';
+// TODO: what are the fallbacks for the relay chain constants?
```

Comment:
If xcm is enabled, there are no fallbacks. You could enable/disable the XCM based on the presence of those fields (logging to console if XCM is disabled bc of this)

Reply:
On this, I'll handle it as a UI task so it won't be in this PR, but I'll run the check in the app and disable xcm if the variables aren't set (similar to how we disable the vault UI).